activesupport/lib/active_support/evented_file_update_checker.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-gem "listen"
+gem "listen", "~> 3.3"
```

Comment:
we might want to use 3.4.1 (or 3.5) as minimum to include this fix: https://github.com/guard/listen/pull/529

Reply:
That's a good point, I updated to be `~> 3.5`